graphql_api/types/coverage_analytics/coverage_analytics.py

```diff
@@ -100,3 +114,229 @@ async def resolve_measurements(
     )
 
     return measurements
+
+
+@coverage_analytics_bindable.field("components")
+@convert_kwargs_to_snake_case
```

Comment:
we actually don't need this decorator anymore! Now that we have this:

https://github.com/codecov/codecov-api/blob/9aa6458ad023fe0ec55993ad9134bbd0af5fb3f7/graphql_api/schema.py#L7

Reply:
errr by this I mean the "@convert_kwargs_to_snake_case" decorator

Reply:
nice, fixed!